diff --git a/node_modules/expo-image-picker/ios/MediaHandler.swift b/node_modules/expo-image-picker/ios/MediaHandler.swift
index a0e3a44..21af462 100644
--- a/node_modules/expo-image-picker/ios/MediaHandler.swift
+++ b/node_modules/expo-image-picker/ios/MediaHandler.swift
@@ -296,20 +296,12 @@ internal struct MediaHandler {
   }
 
   private func getMimeType(from asset: PHAsset?, fileExtension: String) -> String? {
-    let utType: UTType? = if #available(iOS 26.0, *) {
-      asset?.contentType ?? UTType(filenameExtension: fileExtension)
-    } else {
-      UTType(filenameExtension: fileExtension)
-    }
+    let utType = UTType(filenameExtension: fileExtension)
     return utType?.preferredMIMEType
   }
 
   private func getMimeType(from resource: PHAssetResource, fileExtension: String) -> String? {
-    let utType: UTType? = if #available(iOS 26.0, *) {
-      resource.contentType
-    } else {
-      UTType(resource.uniformTypeIdentifier) ?? UTType(filenameExtension: fileExtension)
-    }
+    let utType = UTType(resource.uniformTypeIdentifier) ?? UTType(filenameExtension: fileExtension)
     return utType?.preferredMIMEType
   }
 
